python/cudf/cudf/core/column/categorical.py

```diff
@@ -946,7 +949,7 @@ def to_pandas(self, index: pd.Index = None, **kwargs) -> pd.Series:
             col = self
 
         signed_dtype = min_signed_type(len(col.categories))
-        codes = col.codes.astype(signed_dtype).fillna(-1).to_array()
+        codes = col.codes.astype(signed_dtype).fillna(-1).values_host
```

Comment:
Is this `-1` used to mean the same thing as `_DEFAULT_CATEGORICAL_VALUE`?
```suggestion
        codes = col.codes.astype(signed_dtype).fillna(_DEFAULT_CATEGORICAL_VALUE).values_host
```
Same question here: https://github.com/rapidsai/cudf/blob/f46c423f18a26aef8db3bf62b3e6b10eed015884/python/cudf/cudf/core/column/categorical.py#L1604

Reply:
Good catch, I did a find and replace on the sensible places.